requirements/aws.txt

```diff
@@ -4,9 +4,9 @@
 #
 #    make upgrade
 #
-boto3==1.16.63
+boto3==1.17.7
```

Comment:
Have you looked at the boto change log just in case there are any odd new things in the minor version bump?

Reply:
Yes, they just dropped support for Python 3.4 and 3.5, which we already dropped from testeng-ci earlier in #249.